homeworkk3.py

```diff
@@ -0,0 +1,117 @@
+class Webpush:
+    def __init__(self, platform, optin: bool, global_frequency_capping, start_date, end_date, language, push_type):
+        self.platform = platform
+        self.optin = optin
+        self.global_frequency_capping = global_frequency_capping
+        self.start_date = start_date
+        self.end_date = end_date
+        self.language = language
+        self.push_type = push_type
+
+    @staticmethod
+    def send_push():
+        print("**********\nPush Gönderildi!")
+
+
+class Triggerpush(Webpush):
+    def __init__(self, trigger_page: str, platform, optin: bool, global_frequency_capping, start_date, end_date,
+                 language, push_type):
+        super().__init__(platform, optin, global_frequency_capping, start_date, end_date, language, push_type)
+        self.trigger_page = trigger_page
```

Comment:
parametleri ayırt edebilmek için 19-20 arasına bir boşluk koyalım.Diğerlerinde de düzeltelim.

Reply:
düzeltildi.